### PR TITLE
Some improvements around tooling and diskspace

### PR DESCRIPTION
**Pull request by admd**
# Description

Some improvements have been suggested here. Besides that, also swap memory suggestion is being removed as talking with folks, it seems it doesn't make sense to suggestion swap anymore but rather user should have enough memory.

# Target branches

* Which product version this PR applies to (Uyuni, SUMA 5.0
* Does this PR need to be backported? No

Backport targets (edit as needed):

- master

# Links
- This PR tracks issue #<insert spacewalk issue, if any>
- Related development PR #<insert PR link, if any>

### CODE REVIEW

**Review by mbussolotto — APPROVED** [review]

File: modules/administration/pages/space-management.adoc [hunk 1]
```diff
@@ -29,6 +29,7 @@ For example:
 spacecheck_dirs = /var/lib/pgsql /var/spacewalk /var/cache /srv
 ----
 
+For more info about volumes please visit https://documentation.suse.com/suma/5.0/en/suse-manager/installation-and-upgrade/container-management/persistent-container-volumes.html
```
Comment (keichwa) [inline modules/administration/pages/space-management.adoc:32]:
```suggestion
For more information about volumes, see https://documentation.suse.com/suma/5.0/en/suse-manager/installation-and-upgrade/container-management/persistent-container-volumes.html.
```

File: modules/administration/pages/troubleshooting/tshoot-diskspace.adoc [hunk 2]
```diff
@@ -23,6 +23,11 @@ Running out of disk space can have a severe impact on the {productname} database
 {productname} monitors free space in specific directories, and has configurable alerts.
 For more on space management, see xref:administration:space-management.adoc[].
 
+Generally, container volumes share space with the host filesystem. When the btrfs filesystem becomes full, it can prevent you from deleting anything.
+To resolve this, you can add a small amount of storage to the filesystem using [command]``btrfs device add`` command. This will allow you to delete the necessary files.
+After that, you can remove the additional storage with [command]``btrfs device delete``. Some more troubleshooting information around this topic can be found here https://www.suse.com/support/kb/doc/?id=000018779
+
```
Comment (keichwa) [inline modules/administration/pages/troubleshooting/tshoot-diskspace.adoc:29]: This is just a suggestion.  I'm not sure I completely understood the operations involved and then used wording clear enough.
```suggestion
Generally, container volumes share space with the host filesystem.
When the [systemitem]``btrfs`` filesystem becomes full, it can prevent you from deleting files.
To resolve this, you can add a small amount of storage temporarily to the filesystem using the [command]``btrfs device add`` command.
This will allow you to delete files to gain space for further filesystem maintenance.
When done with the maintenance, you can remove the temporary storage with [command]``btrfs device delete``.
For more information about this topic, see https://www.suse.com/support/kb/doc/?id=000018779.
```

File: modules/administration/pages/troubleshooting/tshoot-diskspace.adoc [hunk 3]
```diff
@@ -23,6 +23,11 @@ Running out of disk space can have a severe impact on the {productname} database
...
+
 You can recover disk space by removing unused software channels.
 For instructions on how to delete vendor channels, see xref:administration:channel-management.adoc[].
 For instructions on how to delete custom channels, see xref:administration:custom-channels.adoc[].
```
Comment (keichwa) [inline modules/administration/pages/troubleshooting/tshoot-diskspace.adoc:33]: Drive-by edit:

```suggestion
* For instructions on how to delete vendor channels, see xref:administration:channel-management.adoc[].
* For instructions on how to delete custom channels, see xref:administration:custom-channels.adoc[].
```

File: modules/installation-and-upgrade/pages/container-management/updating-proxy-containers.adoc [hunk 4]
```diff
@@ -1,6 +1,16 @@
...
+
+. One can do so by running the following command:
++
+----
+transactional-update
+----
+
+. If updates were applied, [literal]``reboot``.
+
+. The {productname} {productnumber} Proxy containers running on [literal]``podman`` can be updated using the following command:
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/container-management/updating-proxy-containers.adoc:13]:
```suggestion
. The {productname} {productnumber} Proxy containers running on [literal]``podman`` can be updated using the following command:
+

```
Without this "connector" adoc tools sometimes end the procedure the following text block.

File: modules/installation-and-upgrade/pages/container-management/updating-server-containers.adoc [hunk 5]
```diff
@@ -1,6 +1,16 @@
...
+
+. One can do so by running the following command:
++
+----
+transactional-update
+----
+
+. If updates were applied, [literal]``reboot``.
+
+. The {productname} {productnumber} Server container can be updated using the following command:
```
Comment (keichwa) [inline modules/installation-and-upgrade/pages/container-management/updating-server-containers.adoc:14]:
```suggestion
+

```

**Review by keichwa — CHANGES_REQUESTED** [review]

File: modules/administration/pages/troubleshooting/tshoot-diskspace.adoc [hunk 6]
```diff
@@ -23,9 +23,14 @@ Running out of disk space can have a severe impact on the {productname} database
...
 
+Generally, container volumes share space with the host filesystem. When the btrfs filesystem becomes full, it can prevent you from deleting anything.
+To resolve this, you can add a small amount of storage to the filesystem using [command]``btrfs device add`` command. This will allow you to delete the necessary files.
+After that, you can remove the additional storage with [command]``btrfs device delete``. Some more troubleshooting information around this topic can be found here https://www.suse.com/support/kb/doc/?id=000018779
+
+
 You can recover disk space by removing unused software channels.
-For instructions on how to delete vendor channels, see xref:administration:channel-management.adoc[].
-For instructions on how to delete custom channels, see xref:administration:custom-channels.adoc[].
+* For instructions on how to delete vendor channels, see xref:administration:channel-management.adoc[].
```
Comment (0rnela) [inline modules/administration/pages/troubleshooting/tshoot-diskspace.adoc:32]:
```suggestion

* For instructions on how to delete vendor channels, see xref:administration:channel-management.adoc[].
```

File: modules/installation-and-upgrade/pages/container-management/updating-proxy-containers.adoc [hunk 7]
```diff
@@ -1,6 +1,18 @@
 = Update containers
```
Comment (0rnela) [inline modules/installation-and-upgrade/pages/container-management/updating-proxy-containers.adoc:1]:
```suggestion
= Update Containers
```

File: modules/installation-and-upgrade/pages/container-management/updating-server-containers.adoc [hunk 8]
```diff
@@ -1,6 +1,17 @@
 = Update containers
```
Comment (0rnela) [inline modules/installation-and-upgrade/pages/container-management/updating-server-containers.adoc:1]:
```suggestion
= Update Containers
```

**Comment by admd** [comment]
Thank you for the review. I have made the suggested changes and squashed some commits.

**Review by keichwa — APPROVED** [review]
Thank you, Abid!
